frontend/src/views/Orders/Orders.scss

```diff
@@ -0,0 +1,12 @@
+@import "../../stylebase/variables";
+@import "../../stylebase/utils";
+
+.Orders {
+  .Orders-heading {
+    padding-top: $padding-wrapper;
+    font-weight: 700;
+  }
+  .Orders-heading-pl {
```

Comment:
The View `withPadding` should fix the missing padding. You then might have to set the margin of the tabs to `(-$padding-wrapper)`

Reply:
the props `withPadding` as a way of distorting my styling. And Negative margin is not capable of given me desired result. It's only resolve to the left, by adding the left space to the right

Reply:
This is screenshot for more clarification on above discussion. 
> withPadding and negative margin
![doubled-right](https://user-images.githubusercontent.com/31921958/81062147-61e41980-8ecd-11ea-936e-83a9fb15e9dc.png)

> with no padding props
![no-withPadding](https://user-images.githubusercontent.com/31921958/81062236-89d37d00-8ecd-11ea-93f3-71228da23b06.png)